.clang-format

```diff
@@ -53,4 +53,5 @@ SpacesInContainerLiterals: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 UseTab: Never
-...
+IndentRequiresClause: false
+RequiresClausePosition: OwnLine
```

Comment:
Sorry but llvm 15 is not available on manjaro for now, preventing others to work on the project.

Reply:
I reformated using clang-format 14